ci

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -exuo pipefail
+
+bundle exec rspec
+#bundle exec rubocop -d
```

Comment:
Unfortunately `rubocop` isn't available on master yet so this doesn't run. The comment was an unfortunate median between adding Rubocop or skipping it for this PR (I've now removed the comment, will follow up).